packages/contentful--app-scripts/lib/utils.js

```diff
@@ -73,7 +79,7 @@ const selectFromList = async (list, message, cachedOptionEnvVar) => {
   }
 };
 
-function getActionsManifest() {
+function getEntityFromManifest(type) {
```

Comment:
Alternatively, we could always parse both and return an object with two props 🤷

Reply:
But this way the function is smaller 😛